cmd/brew-bootstrap-rbenv-ruby

```diff
@@ -49,9 +49,10 @@ if ! rbenv version-name &>/dev/null; then
   export RUBY_CONFIGURE_OPTS="--with-openssl-dir=$HOMEBREW_PREFIX/opt/openssl"
 
   rbenv install --skip-existing "$RUBY_DEFINITION"
-  if [ "$(rbenv exec ruby --version)" != "$(ruby --version)" ]; then
-    abort 'Error: add `eval "$(rbenv init -)"` to the end of your .bash_profile/.zshrc!'
-  fi
+fi
+
+if [ "$(rbenv exec ruby --version)" != "$(ruby --version)" ]; then
+  abort 'Error: add `eval "$(rbenv init -)"` to the end of your .bash_profile/.zshrc!'
```

Comment:
I personally think it's worth just stopping at this point as notifying them of an action they need to take will result in some people missing that action and having to deal with the confusing bugs that result. I'd be up for improving this message based on looking at `$SHELL`, though (you can even add 🐟 )